terraso_backend/apps/core/permission_rules.py

```diff
@@ -103,6 +103,16 @@ def allowed_to_update_preferences(user, user_preferences):
     return user_preferences.user.id == user.id
 
 
+@rules.predicate
+def is_site_creator(user, site):
```

Comment:
Suggestion: Follow the same naming convention as the other rules `allowed_to_...`